src/components/features/cart/pages/home.hbs

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
this file should ideally contain your body section, excluding header and footer.

Move the other code in a generic base.hbs file with head, header and footer in it.